test_massive_iterator.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from iterator_massive import MassiveIterator
+
+
+class TestMassiveIterator(unittest.TestCase):
+    @unittest.expectedFailure
```

Comment:
These tests just repeat `TestMassiveGenerator`, could we get rid of code duplication here?